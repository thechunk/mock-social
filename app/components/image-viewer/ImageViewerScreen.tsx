import * as React from 'react';
import {PureComponent} from "react";
import ImageViewer from './ImageViewer';
import {NavigationInjectedProps} from "react-navigation";
import withNavigationParamsAsProps from "../../hoc/withNavigationParamsAsProps";
import styles from './styles';

class ImageViewerScreen extends PureComponent<IImageViewerScreen & NavigationInjectedProps> {
    render() {
        return <ImageViewer url={this.props.hydrate.url} style={styles.viewer} />;
    }
}

export default withNavigationParamsAsProps(ImageViewerScreen);