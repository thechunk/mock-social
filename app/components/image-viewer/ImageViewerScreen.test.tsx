import * as React from 'react';
import {shallow} from "enzyme";
import {ImageViewerScreen} from "./ImageViewerScreen";

describe('ImageViewerScreen', () => {
    it('renders correctly', () => {
        expect(shallow(<ImageViewerScreen hydrate={{url: 'https://url'}} />)).toMatchSnapshot();
    });
});