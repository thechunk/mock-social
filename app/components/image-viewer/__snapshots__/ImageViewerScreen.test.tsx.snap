// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageViewerScreen renders correctly 1`] = `
<_class
  style={
    Object {
      "flex": 1,
    }
  }
  url="https://url"
/>
`;
